server.js

```diff
@@ -102,6 +102,14 @@ server.register([
   server.views(require('./src/views'))
 })
 
+server.ext('onPreResponse', function (request, reply) {
+  if (request.response.isBoom) {
```

Comment:
More a question as I don't understand how this function works. Will this just send 404 errors to the 404 page, or will all/other errors get directed there as well?

Reply:
Well spotted, I have changed it so that only 404 errors will result in a redirection to the /page-not-found route.